components/Main/Card.styled.js

```diff
@@ -2,36 +2,27 @@ import styled from "styled-components";
 import { BsSearch } from "react-icons/bs";
 import { MdDelete } from "react-icons/md";
 import { MdEdit } from "react-icons/md";
```

Comment:
You could combine these imports in one. 
```suggestion
import { MdDelete, MdEdit } from "react-icons/md";
```